app/processing/quarantine-file.js

```diff
@@ -2,7 +2,6 @@ const storage = require('../storage')
 const sendBatchQuarantineEvent = require('../event/send-batch-quarantine-event')
 
 const quarantineFile = async (filename) => {
-  console.error(`Quarantining ${filename}, failed to parse file`)
```

Comment:
Duplicate log line.  Call functions log the context of quarantine so adds nothing